torchci/lib/JobClassifierUtil.ts

```diff
@@ -1,115 +1,187 @@
 import { GroupedJobStatus, JobStatus } from "components/GroupJobConclusion";
 import { GroupData, RowData } from "./types";
 
+const GROUP_MEMORY_LEAK_CHECK = "Memory Leak Check"
```

Comment:
Nit: what do you think of adding a prefix to the variable name here as a cue to their relative order?  For example, `const 000_GROUP_LINT = "Lint"`, then `const 001_GROUP_LINUX = "Linux"`.  The names look kind of ugly though, so ...

Reply:
I'd prefer to keep it easy for anyone to reorder the groups later on, either as we add more groups or if someone decides a different grouping would be more optimal.  The prefixes would make that harder, and the group regexes would also then have the group names appear in seemingly a random order lower down